hudi-utilities/src/test/java/org/apache/hudi/utilities/TestHiveIncrementalPuller.java

```diff
@@ -51,11 +53,16 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-public class TestHiveIncrementalPuller {
+public class TestHiveIncrementalPuller extends UtilitiesTestBase {
 
   private HiveIncrementalPuller.Config config;
   private String targetBasePath = null;
 
+  @BeforeAll
+  public static void setupOnce() throws Exception {
+    initTestServices();
+  }
```

Comment:
but this did not need utilities test base, what test resources we are adding here?

Reply:
Added to the wrong file, I'll remove this